rubo_claus.gemspec

```diff
@@ -14,6 +14,9 @@ Gem::Specification.new do |s|
   s.test_files    = s.files.grep(%r{^(test|spec|features)/})
   s.require_paths = ["lib"]
   s.license 			= 'MIT'
+  s.homepage      = 'https://github.com/mojotech/rubo_claus'
```

Comment:
Please change this to the link I provided to you :-) http://mojotech.github.io/rubo_claus/

Will this be reflected in the gem page?